src/components/Button/index.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import './ButtonStyles.css';
+
+const Button = ({ click, className, iconClassName }) => <div onClick={click} className={className}><i className={iconClassName}></i></div>
```

Comment:
not critical, but let's name all props related to events as `on<Action>` in a more React-way. For this case it could be:
```suggestion
const Button = ({ onClick, className, iconClassName }) => <div onClick={click} className={className}><i className={iconClassName}></i></div>
```

Reply:
Got you Sr. Solved